spec/policies/cancellation_policy_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+RSpec.describe PairRequest::CancellationPolicy do
+  subject { described_class }
+
+  let(:user) { build(:user) }
+  let(:author) { build(:user) }
+  let(:pending_pair_request) { build(:pair_request, author:) }
```

Comment:
the more I think about this, the more I'm not super keen on how many `let` statements I'd like to leverage here, but am happy to leave be or update

Reply:
I think this is fine.